docs/howto/custom-template-tags.txt

```diff
@@ -327,6 +327,23 @@ Template filter code falls into one of two situations:
    handle the auto-escaping issues and return a safe string, the
    ``is_safe`` flag won't change anything either way.
 
+   This pattern of using conditional_escape if autoescape is enabled is so
+   common that there is a decorator
+   .. method:: django.template.defaultfilters.autoescape_aware() which can be
```

Comment:
I suspect the docs failure is because this Sphinx link is broken:

```suggestion
   :func:`~django.template.defaultfilters.autoescape_aware` which can be
```

**Before**:
<img width="611" alt="Screen Shot 2021-10-04 at 8 04 26 PM" src="https://user-images.githubusercontent.com/38668450/135940408-c6b71950-ef24-439c-b831-9fdaec7a7364.png">

Reply:
Hey @jacobtylerwalls, thanks for the suggestion, but the tests still error out as they cannot locate the function. :confused:

Reply:
Failure goes away when adding `@stringfilter` like the other filters. Is that appropriate here? I haven't looked carefully.

Reply:
Rats. I thought the failure resolved when I built your PR locally. Looks like that was wrong, sorry.

Reply:
It's fine. Do tell me if you get a hunch about how to fix this :).